MdeModulePkg/Include/UniversalPayload/SerialPortInfo.h

```diff
@@ -19,6 +19,7 @@ typedef struct {
   BOOLEAN                             UseMmio;
   UINT8                               RegisterStride;
   UINT32                              BaudRate;
+  UINT32                              ClockRate;

```

Comment:
Follow up backward compatible for structure, I want to recommend you put this code from line 22 to line 24